graylog2-server/src/main/resources/swagger/index.html.template

```diff
@@ -1,98 +1,98 @@
-    <span class="subtitle">REST API browser</span>
-    <form id="api_selector">
-      <div class="input">
-        <input type="text" name="user" id="input_apiUser" placeholder="Username"/>
-      </div>
-      <div class="input">
-        <input type="password" name="password" id="input_apiPassword" placeholder="Password"/>
-      </div>
-    </form>
-  </div>
-</div>
-
-<div id="swagger-ui-container" class="swagger-ui-wrap">
-</div>
-
-<div id="credits">
-    This browser is a customized version of the great
-    <a href="https://developers.helloreverb.com/swagger/" target="_blank">Swagger</a> project.
-</div>
-
-</body>
-
-</html>
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Graylog REST API browser</title>
+  <link href='//fonts.googleapis.com/css?family=Droid+Sans:400,700' rel='stylesheet' type='text/css'/>
+  <link href='${apiPrefix}api-browser/css/highlight.default.css' media='screen' rel='stylesheet' type='text/css'/>
+  <link href='${apiPrefix}api-browser/css/screen.css' media='screen' rel='stylesheet' type='text/css'/>
+  <script src='${apiPrefix}api-browser/lib/shred.bundle.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/jquery-1.8.0.min.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/jquery.slideto.min.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/jquery.wiggle.min.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/jquery.ba-bbq.min.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/handlebars-1.0.0.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/underscore-min.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/backbone-min.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/swagger.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/swagger-ui.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/highlight.7.3.pack.js' type='text/javascript'></script>
+  <script src='${apiPrefix}api-browser/lib/sha256.js'></script>
+  <script type="text/javascript">
+    var port = window.location.port;
+    if (port == '') {
+      if (window.location.protocol == 'http') {
+        port = 80;
+      } else if (window.location.protocol == 'https') {
+        port = 443;
+      }
+    }
+    var apiTarget = window.location.protocol + "//" + window.location.hostname + ":" + port + "${apiPrefix}";
+    $(function () {
+      window.swaggerUi = new SwaggerUi({
+      url: apiTarget + "api-docs",
+      dom_id: "swagger-ui-container",
+      supportedSubmitMethods: ['get', 'post', 'put', 'delete'],
+      onComplete: function(swaggerApi, swaggerUi){
+
+        // We are taking the base path of the system API here. This will break if you should ever rename it lol.
+        if(swaggerApi.System && !apiTarget.startsWith(swaggerApi.System.basePath))   {
```

Comment:
I think `startsWith` is not supported in older browsers. Probably better to use
`apiTarget.lastIndexOf(swaggerApi.System.basePath, 0) === 0`